LifeTimeExample.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.24;
+
+contract LifeTimeExample {
+    address public owner;
+    uint storedData;
+    bool public freeze;
+
+    constructor() public {
+        owner = msg.sender;
+        freeze = false;
+    }
+
+    modifier CheckFreeze {
+        require(freeze == false, "This function is freezed.");
+        _;
+    }
+
+    function setFreeze() public {
+        freeze = true;
+    }
+
+    function setUnfreeze() public {
+        freeze = false;
+    }
+
+    function set(uint x) CheckFreeze public {
+        storedData = x;
+    }
+
+    function get() view public returns (uint) {
```

Comment:
Follow the style guide:
This fuction should at the next of `kill`

Reply:
Done!